Makefile

```diff
@@ -1,31 +1,27 @@
-help: ## Display this help screen
-	@grep -h -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-15s\033[0m %s\n", $$1, $$2}'
+.PHONY: wasm contracts keys circuits abi test run help
```

Comment:
The phony should be at the end of the makefile, for both consistency and clarity (you don't refer to recipes that you didn't write yet). Any particular reason why you moved on the top?

Reply:
I believe its usually in the top - as in the GNU examples:

https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

Reply:
Only when you have small recipe, yes, so it doesn't matter since you get a glance of it without scrolling, in that case you put it either before or after the recipe is using. With more complex / longer Makefile (such as ours) it usually at the end, after the definition of all the recipe previously defined. For example: https://github.com/PyO3/tokio/blob/master/Makefile

Also, since this was already the convention we used I don't see the point to change it unless it causes issues.

Reply:
I don't think that was a convention for us. The oldest one is on the top, too:
https://github.com/dusk-network/dusk-blockchain/blob/master/Makefile

Seems the recent commits are going in a different direction.

Reply:
For the core team the `.PHONY` was always at the bottom and I believe the majority of our makefile are still like that. At the end it's a convention (like having a function call after the definition of the function instead of before, even if it's hoisted), and since this Makefile already have the `.PHONY` at the bottom I don't see any reason to change it, especially if it's diverging from the majority of Makefile we have in core.